DevCycleTests/Models/DVCVariableTests.swift

```diff
@@ -144,6 +223,29 @@ class DVCVariableTests: XCTestCase {
         XCTAssertTrue(variable.isDefaulted)
     }
     
+    func testReturnsDefaultValueIfJSONObjectDoesntMatch() throws {
+        let data = """
+        {
+            "_id": "variable_id",
+            "key": "my_key",
+            "type": "JSON",
+            "value": {
+                "key1": "value1",
+                "key2": {
+                    "nestedKey1": 610
+                }
+            }
+        }
+        """.data(using: .utf8)!
+        let variableDict = try JSONSerialization.jsonObject(with: data, options: .fragmentsAllowed) as! [String: Any]
+        let variableFromApi = try Variable(from: variableDict)
+        let defaultValue = ["key1": "value2"]
```

Comment:
shouldn't it not be possible to pass arrays? or are you allowing that

Reply:
That's a `Dictionary<String, String>` welcome to swift :P

Reply:
oh lol